consensus/service/README.md

```diff
@@ -173,7 +173,7 @@ SGX_MODE=HW IAS_MODE=DEV \
 Alternatively, if the binary has already been built, you can run:
 
 ```
-./target/release/consensus-service --client-responder-id \
+${CARGO_TARGET_DIR}/release/consensus-service --client-responder-id \
```

Comment:
```suggestion
${CARGO_TARGET_DIR:-target}/release/consensus-service --client-responder-id \
```

Reply:
Should be addressed now, the suggestion was slightly modified to accomodate for `./target`